README.md

```diff
@@ -98,26 +99,32 @@ The event name and what's in the `data` for each event handler:
   * `event` The raw event
   * `senderId` The ID of the sender
   * `session` [A Session object](#the-session-object) you can mutate
-  * `text` Message content, `event.message.text` for text events
+  * `payload` Message content, `event.message.text` for text events
```

Comment:
I thought that's what was understood as the necessary change to make this consistent... if not, then what did you mean here:

> waiting on the updates to plain old message.text to match postback and quickReply

?